# What does the following error message tell you?
#
# ArgumentError: wrong number of arguments (1 for 2)
#   from (irb):1:in `calculate_product'
#   from (irb):4
#   from /Users/username/.rvm/rubies/ruby-2.0.0-p353/bin/irb:12:in `<main>'
#
#
#   The following tells me that there the wrong number of arguments have been passed calculate_product(a,b) into the method for example calculate_product(1) when it should be calculate_product(1,2) 
